src/parallel/MPI_TIMED/ProcessTimer.h

```diff
@@ -0,0 +1,120 @@
+/*
+ * ProcessTimer.h
+ *
+ *  Created on: Apr 12, 2020
+ *      Author: jeremyharisch
+ */
+
+#ifndef MARDYN_PROCESSTIMER_H
+#define MARDYN_PROCESSTIMER_H
+#define _BSD_SOURCE
```

Comment:
why do you need this?
probably, we don't like this ;)

Reply:
Do you mean the creator comment? I'll remove it